### PR TITLE
Add bat script to run integration tests

### PR DESCRIPTION
**Pull request by se7entyse7en**
As part of #168.

This adds a bat script to run integration tests as we did for [engine](https://github.com/src-d/engine-deprecated/blob/master/run-integration-tests.bat).

Integration tests can then be run as follows:
```
PS C:\Users\Windows10\sourced-ce> $env:SOURCED_GITHUB_TOKEN="<github token>"
PS C:\Users\Windows10\sourced-ce> .\run-integration-tests.bat
```

### CODE REVIEW

File: run-integration-tests.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:: we can't use makefile for windows because it depends on CI makefile which depends on shell
+
+:: compile sourced-ce
+if not exist build mkdir build
```
Comment (dpordomingo) [inline run-integration-tests.bat]: is not `build/sourced-ce_windows_amd64` automatically created when runnuning `go build -o build/sourced-ce_windows_amd64/sourced.exe ./cmd/sourced` on windows? In Ubuntu it does.

**Review by dpordomingo — COMMENTED** [review]
LGTM but I have a question

File: run-integration-tests.bat (hunk 1 above)
Comment (se7entyse7en) [reply run-integration-tests.bat]: You're right, the dirs are created by `go build` itself. Addressed here: https://github.com/src-d/sourced-ce/pull/171/commits/e3b9e9e348fdf5aae52e629b8ed1a11c73053874.

**Comment by se7entyse7en** [comment]
I also changed the new line encoding to `carriage return + line feed` given that it is a Windows file here: https://github.com/src-d/sourced-ce/pull/171/commits/a58b5531f9669dff85af470a4574324e787ab830.

**Review by dpordomingo — APPROVED** [review]

**Review by smacker — APPROVED** [review]

**Review by carlosms — APPROVED** [review]

**Pull request merged**